### PR TITLE
feature(prettier): apply prettier to src/lib/helpers

### PR DESCRIPTION
**Pull request by MFrank84**
@plentymarkets/team-terra

### CODE REVIEW

File: src/lib/helpers/tests/number.helper.spec.ts [hunk 1]
```diff
@@ -26,11 +23,10 @@ describe('Helper: NumberHelper', () =>
         expect(NumberHelper.round(-5.432, 2)).toEqual(-5.43);
     });
 
-    it('should round edge cases', () =>
-    {
+    it('should round edge cases', () => {
         expect(NumberHelper.round(5, 0)).toEqual(5);
         expect(NumberHelper.round(5, 30)).toEqual(5);
-        expect(NumberHelper.round(5.00000, 5)).toEqual(5);
+        expect(NumberHelper.round(5.0, 5)).toEqual(5);
```
Comment (dantrau) [inline src/lib/helpers/tests/number.helper.spec.ts:29]: das muss so bleiben

Comment (MFrank84) [reply src/lib/helpers/tests/number.helper.spec.ts:29]: Vom Prinzip her schon, aber bei genauerer Betrachtung wird der Helper mit mehreren Nachkommastellen schon gut getestet und viele 0'en sind halt auch nicht so sinnvoll. Ich würde vorschlagen der Test `expect(NumberHelper.round(5.0, 5)).toEqual(5);` wird hoch zu den anderen 1-stelligen Tests geschoben

Comment (dantrau) [reply src/lib/helpers/tests/number.helper.spec.ts:29]: ja, dann bin ich dafür die Expectation zu entfernen. das darunter reicht auch schon
